src/components/buttons/save-button/save-button.test.js

```diff
@@ -46,7 +46,7 @@ describe('save button tests', () => {
     component.find('button').simulate('click');
   });
   it('should exist and have value', () => {
-    expect(component.exists());
+    expect(component.exists()).toBeTruthy();
```

Comment:
it will be better to delete this - expect(component.exists()).toBeTruthy(); - because here we already have one expect and it checks if the component exists.